app/assets/config/manifest.js

```diff
@@ -1,2 +1,3 @@
 //= link_tree ../images
-//= link_directory ../stylesheets .css
+//= link application.css
```

Comment:
I had to redirect the complier to look at the mains css file, rather than looking in the stylesheets directory for all of the css files. This was causing an undefined variable error beforehand when the sass variable was not being defined as the file was not being read.